### ISSUE TITLE
PHP Fatal error:  Uncaught OutOfBoundsException: Package "spatie/ray" is not installed

### ISSUE THREAD

**Issue by slaFFik** [spatie/wordpress-ray#12]
PHP error:

```
[13-Jan-2021 11:20:34 UTC] PHP Fatal error:  Uncaught OutOfBoundsException: Package "spatie/ray" is not installed in C:\OpenServer\domains\wordpress\wp-content\plugins\custom-plugin\vendor\composer\InstalledVersions.php:186
Stack trace:
#0 C:\OpenServer\domains\wordpress\wp-content\plugins\wordpress-ray\vendor\spatie\ray\src\Ray.php(351): Composer\InstalledVersions::getVersion()
#1 C:\OpenServer\domains\wordpress\wp-content\plugins\wordpress-ray\src\Ray.php(104): Spatie\WordPressRay\Spatie\Ray\Ray->sendRequest()
#2 C:\OpenServer\domains\wordpress\wp-content\plugins\wordpress-ray\vendor\spatie\ray\src\Ray.php(320): Spatie\WordPressRay\Ray->sendRequest()
#3 C:\OpenServer\domains\wordpress\wp-content\plugins\wordpress-ray\vendor\spatie\ray\src\helpers.php(34): Spatie\WordPressRay\Spatie\Ray\Ray->send()
#4 C:\OpenServer\domains\wordpress\wp-content\plugins\custom-plugin\src\CustomPlugin.php(279): ray()
#5 C:\OpenServer\domains\wordpress\wp-content\plugins\custom-plugin\src\CustomPlugin.php(219): CustomPlugin\CustomPlugin->includes_m in C:\OpenServer\domains\wordpress\wp-content\plugins\custom-plugin\vendor\composer\InstalledVersions.php on line 186
```

Steps:
1. Download the repo as a zip.
2. Upload the zip to WordPress as a plugin.
3. Activate the plugin.
4. Put the `ray($this);` call inside some other plugin, that uses composer for its dependencies as well.
5. Reload the WordPress page - get the fatal error.

I tried to run `composer install` inside the `wordpress-ray` directory as well, just in case. Imposter and dumpautoload successfully run, nothing's changed.

Using WordPress 5.6, PHP 7.4, Composer v2, wordpress-ray master branch as of now.

**Comment by slaFFik** [comment]
Checked the just-released v0.0.9 from the repo as well.

**Comment by slaFFik** [comment]
Same issue with just released plugin on wp.org: https://wordpress.org/plugins/spatie-ray/

Also, because the released version is having WordPress 5.5 defined as tested up to in the `readme.txt` - you can't easily find the plugin now.

**Comment by freekmurze** [comment]
Thanks for reporting this, I've tried to fix it in 0.12.0, hopefully that one works.

**Comment by slaFFik** [comment]
@freekmurze I'm getting now this fatal:
```
Parse error: syntax error, unexpected 'public' (T_PUBLIC) in C:\OpenServer\domains\wordpress\wp-content\plugins\wordpress-ray\vendor\spatie\ray\src\Ray.php on line 376
```
for 0.12 tag

**Comment by slaFFik** [comment]
@freekmurze `sendRequest()` method doesn't have a closing `}`.

**Comment by slaFFik** [comment]
Adding that } fixes both of these fatals and your plugin starts to work and send data to the Ray app.

**Comment by freekmurze** [comment]
I think this is fixed in 0.13.0? Could you confirm that?

Thanks btw for helping me test this 👍

**Comment by slaFFik** [comment]
@freekmurze Yes, 0.13 tag (its attached `spatie-ray.zip`) is working fine for me now. I think we can close this issue.
